iosHackthonApp/Modules/Features/Login/.gitignore

```diff
@@ -0,0 +1,5 @@
+.DS_Store
+/.build
+/Packages
```

Comment:
Sorry, I should have said earlier - you can delete this file

Reply:
Done